lib/errors/silent.js

```diff
@@ -3,9 +3,12 @@
 var SilentError = require('silent-error');
 var deprecate   = require('../utilities/deprecate');
 
+// Get the call stack so we can let the user know what module is using the deprecated function.
+var stack = new Error().stack;
+
 Object.defineProperty(module, 'exports', {
   get: function () {
-    deprecate('`ember-cli/lib/errors/silent.js` is deprecated, use `silent-error` instead.', true);
+    deprecate("`ember-cli/lib/errors/silent.js` is deprecated, use `silent-error` instead.  Module: \n" + stack.toString(), true);
```

Comment:
Here's the output:

```
DEPRECATION: `ember-cli/lib/errors/silent.js` is deprecated, use `silent-error` instead. Module: 
Error
    at Module.Object.defineProperty.get [as exports] (/home/ryan/RxTrax/rxtrax-n/node_modules/ember-cli/lib/errors/silent.js:9:17)
    at Function.Module._load (module.js:411:16)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/ryan/RxTrax/rxtrax-n/node_modules/ember-deploy-s3-index/lib/s3-adapter.js:5:19)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
```

The ` Object.<anonymous> (/home/ryan/RxTrax/rxtrax-n/node_modules/ember-deploy-s3-index/lib/s3-adapter.js:5:19)` cued me in on what package was causing the deprecation notice.

What may be better is to do is a text replace for the stack output: `Error at` replaced with `Call stack`